app/graphql/types/runtime_function_definition_type.rb

```diff
@@ -13,6 +13,8 @@ class RuntimeFunctionDefinitionType < Types::BaseObject
           null: true,
           description: 'Function definitions of the Node Function'
 
+    field :identifier, String, null: false, description: 'Identifier of the Node Function'
```

Comment:
The property is named different on the model.
```suggestion
    field :identifier, String, null: false, description: 'Identifier of the Node Function', method: :runtime_name
```